website/docs/guides/docs/docs-introduction.md

```diff
@@ -23,94 +23,4 @@ Your site's documentation is organized by four levels, from lowest to highest:
 
-## Docs-only mode {#docs-only-mode}
-
-A freshly initialized Docusaurus site has the following structure:
-
-```
-example.com/                                -> generated from `src/pages/index.js`
-
-example.com/docs/intro                      -> generated from `docs/intro.md`
-example.com/docs/tutorial-basics/...        -> generated from `docs/tutorial-basics/...`
-...
-
-example.com/blog/2021/08/26/welcome         -> generated from `blog/2021-08-26-welcome/index.md`
-example.com/blog/2021/08/01/mdx-blog-post   -> generated from `blog/2021-08-01-mdx-blog-post.mdx`
-...
-```
-
-All docs will be served under the subroute `docs/`. But what if **your site only has docs**, or you want to prioritize your docs by putting them at the root?
-
-Assume that you have the following in your configuration:
-
-```js title="docusaurus.config.js"
-module.exports = {
-  // ...
-  presets: [
-    '@docusaurus/preset-classic',
-    {
-      docs: {
-        /* docs plugin options */
-      },
-      blog: {
-        /* blog plugin options */
-      },
-      // ...
-    },
-  ],
-};
-```
-
-To enter docs-only mode, change it to like this:
-
-```js title="docusaurus.config.js"
-module.exports = {
-  // ...
-  presets: [
-    '@docusaurus/preset-classic',
-    {
-      docs: {
-        // highlight-next-line
-        routeBasePath: '/', // Serve the docs at the site's root
-        /* other docs plugin options */
-      },
-      // highlight-next-line
-      blog: false, // Optional: disable the blog plugin
-      // ...
-    },
-  ],
-};
-```
-
-Note that you **don't necessarily have to give up on using the blog** or other plugins; all that `routeBasePath: '/'` does is that instead of serving the docs through `https://example.com/docs/some-doc`, they are now at the site root: `https://example.com/some-doc`. The blog, if enabled, can still be accessed through the `blog/` subroute.
-
-Don't forget to put some page at the root (`https://example.com/`) through adding the front matter:
-
-```md title="docs/intro.md"
----
-# highlight-next-line
-slug: /
----
-
-This page will be the home page when users visit https://example.com/.
-```
-
-:::caution
-
-If you added `slug: /` to a doc to make it the homepage, you should delete the existing homepage at `./src/pages/index.js`, or else there will be two files mapping to the same route!
-
-:::
-
-Now, the site's structure will be like the following:
-
-```
-example.com/                       -> generated from `docs/intro.md`
-example.com/tutorial-basics/...    -> generated from `docs/tutorial-basics/...`
-...
-```
-
-:::tip
-
-There's also a "blog-only mode" for those who only want to use the blog feature of Docusaurus 2. You can use the same method detailed above. Follow the setup instructions on [Blog-only mode](../../blog.mdx#blog-only-mode).
-
-:::
+You can have a [Docs-only mode](./docs-only-mode.md).
```

Comment:
weird way to introduce this docs only mode? why is this item separated from others on the list above?